src/index.ts

```diff
@@ -132,12 +145,15 @@ const createDatabase = (
 
 const createCollection = (
   collectionName: string,
-  partitionKey: string
+  partitionKey: string,
+  // eslint-disable-next-line @typescript-eslint/no-explicit-any
+  indexingPolicy?: any
```

Comment:
Use in here a specific type instead of `any` (`IndexingPolicy` from `@azure/cosmos` should be perfect)
```suggestion
  indexingPolicy?: IndexingPolicy
```

Reply:
Done in https://github.com/pagopa/io-fixtures/pull/15/commits/2c1b30dc071277efe1b549d4e13477497b16675a